### PR TITLE
added back button ability in demo

### PR DESCRIPTION
**Pull request by kulothunganug**
### Description of Changes
* get back to home screen instead of not to exit when you click the back button on android

### CODE REVIEW

File: demos/kitchen_sink/main.py [hunk 1]
```diff
@@ -187,6 +188,11 @@ def show_demo_shrine(interval):
         instance.add_widget(box)
         Clock.schedule_once(show_demo_shrine, 1)
 
+
+    def on_key(self, window, key, *args):
+        if key == 27:  # the back key
+            Clock.schedule_once(self.back_to_home_screen)
+
```
Comment (HeaTTheatR) [inline demos/kitchen_sink/main.py]: Without a `return True` instruction, this method will not work.

File: demos/kitchen_sink/main.py [hunk 2]
```diff
@@ -15,6 +15,7 @@
 from libs.baseclass.list_items import (  # NOQA: F401
     KitchenSinkOneLineLeftIconItem,
 )
+from kivy.clock import Clock
```
Comment (HeaTTheatR) [inline demos/kitchen_sink/main.py]: Please follow the standards for importing modules: first, modules of the standard library are imported, then modules and external packages, and at the end - your own modules and packages.

Снимок экрана 2020-11-11 в 09 07 01

**Review by HeaTTheatR — CHANGES_REQUESTED** [review]

File: demos/kitchen_sink/main.py (hunk 2 above)
Comment (kulothunganug) [reply demos/kitchen_sink/main.py]: bro i had used `isort` for sorting modules

Comment (HeaTTheatR) [reply demos/kitchen_sink/main.py]: As you can see, `isort` is not working correctly.

Comment (kulothunganug) [reply demos/kitchen_sink/main.py]: > bro i had used `isort` for sorting modules

mine problem sorry 🙏

File: demos/kitchen_sink/main.py [hunk 3]
```diff
@@ -8,6 +8,10 @@
 from kivy.factory import Factory
 from kivy.lang import Builder
 from kivy.loader import Loader
+from kivymd import images_path
+from kivymd.app import MDApp
+from kivymd.uix.expansionpanel import MDExpansionPanel, MDExpansionPanelOneLine
+
```
Comment (HeaTTheatR) [inline demos/kitchen_sink/main.py:14]: Each import block must be indented.

File: demos/kitchen_sink/main.py (hunk 2 above)
Comment (kulothunganug) [reply demos/kitchen_sink/main.py]:
```
Folder Tree
KivyMD (Cloned)
|____demos
|____docs
|____kivymd
```
when using `python3.9 -m isort demos/kitchen_sink/main.py`

There are no lint errors in PR but `isort` looks the kivymd folder as a local module
```
Folder Tree
KivyMD (Cloned)
|____demos
|____docs
```
By deleting the kivymd folder the `isort` works correctly as u expected but lint error appears in PR

File: demos/kitchen_sink/main.py (hunk 3 above)
Comment (kulothunganug) [reply demos/kitchen_sink/main.py:14]: sorry bro I can't understand what are you telling

File: demos/kitchen_sink/main.py (hunk 2 above)
Comment (HeaTTheatR) [reply demos/kitchen_sink/main.py]: I do not understand what the problem is, importing modules according to `PEP8`?

ezgif-4-f1682d9a805e

**Pull request closed without merging**